tox.ini

```diff
@@ -24,6 +24,8 @@ format = pylint
 [testenv:syntax]
 deps = 
     flake8
+    # NOTE(retr0h): We specifically pin to a known version so formatting
+    # is consistent across developer/PR.
     yapf==0.10.0
```

Comment:
People don't know they need to install yapf?  It installs yapf for them.  No action on users part.